### ISSUE TITLE
Traceback error in pypresence.py

### ISSUE THREAD

**Issue by M4cs** [qwertyquerty/pypresence#51]
This is the error I get when trying to run an RPC Client pypresence script:
```
Traceback (most recent call last):
  File "windows.py", line 2, in <module>
    from pypresence import presence
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\pypresence\__init__.py", line 7, in <module>
    from .activity import Activity
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\pypresence\activity.py", line 4, in <module>
    from .presence import Presence
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\pypresence\presence.py", line 51
    super().__init__(*args, **kwargs, async=True)
                                          ^
SyntaxError: invalid syntax
```

Not sure what this means tbh

**Comment by qwertyquerty** [comment]
Python version?

**Comment by M4cs** [comment]
3.7 
KnownError told me to roll back to 3.6 but also said they tested on 3.7 and it was fine

**Comment by qwertyquerty** [comment]
Show your code, or send a link

**Comment by LewdNeko** [comment]
@M4cs your second error is that you haven't imported time. Or if you have, you imported it under a different name.
The 3.7 error will be fixed

**Comment by LewdNeko** [comment]
All fixed. Not pushed to pypi - install with `pip install https://github.com/qwertyquerty/pypresence/archive/master.zip --upgrade` for 3.7 fixes.

**Comment by qwertyquerty** [comment]
This will be available in 3.2.2 once I push that tonight
